mock-api.js

```diff
@@ -0,0 +1,73 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const app = express();
+const port = 8080;
+
+const responseDelay = 300;
+const alfabet = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ';
+
+const formaterDato = dato =>
+    String(dato.getDate()).padStart(2, '0') +
+    String(dato.getMonth()).padStart(2, '0') +
+    dato.getYear();
+
+const skapTilfeldigDato = (start, end) =>
```

Comment:
Hvor mye verdi får man av å ha tilfeldig generert testdata? Selve mockingen trenger vi vel bare for å få hente noe lokalt og vise dataen.
Koden blir kortere og lettere om man hardkoder alle testdata.

Reply:
Jeg tenker det blir lettere å teste appen med ulike kandidater når man bare kan generere opp folk man vet er gyldige, blir fort slitsomt å lage nye kandidater manuelt. Men om det bare blir rusk i koden kan man jo trekke det ut i et script og copy-paste resultatet inn i mock-fila, og enten ha det i en scripts/-mappe på repoet eller bare bruke den lokalt?